<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvoiMessageDialog</class>
 <widget class="QDialog" name="EnvoiMessageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diffuser un message</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>SÃ©lectionnez votre message :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="messageCBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Diffuser le message dans :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minutesSpinBox">
         <property name="maximum">
          <number>59</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>minutes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="sendButton">
         <property name="text">
          <string>Diffuser</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeSendButton">
         <property name="text">
          <string>Diffuser et fermer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Fermer sans envoyer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minutesSpinBox</tabstop>
  <tabstop>sendButton</tabstop>
  <tabstop>closeSendButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>EnvoiMessageDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeSendButton</sender>
   <signal>clicked()</signal>
   <receiver>EnvoiMessageDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
